AltStore/Settings/SettingsViewController.swift

```diff
@@ -178,7 +178,7 @@ private extension SettingsViewController
             }
             else
             {
-                settingsHeaderFooterView.secondaryLabel.text = NSLocalizedString("Receive access to beta versions of AltStore, Delta, and more by becoming a patron.", comment: "")
+                settingsHeaderFooterView.secondaryLabel.text = NSLocalizedString("Don't Touch this button above =) ", comment: "")
```

Comment:
Wait, why don’t you just remove it entirely?

Reply:
Wasn't sure if it would add problems as I only use actions to compile and edit via vscode. So wasn't sure if I should just remove it. I could change that but maybe someone is wanting some different info in it?

Reply:
The Patreon stuff is being addressed in @bogotesr ’s PR, so I would actually just remove it here to prevent conflicts